rpc/types.go

```diff
@@ -65,7 +65,7 @@ const (
 	FinalizedBlockNumber = BlockNumber(-3)
 	PendingBlockNumber   = BlockNumber(-2)
 	LatestBlockNumber    = BlockNumber(-1)
-	EarliestBlockNumber  = BlockNumber(0)
+	EarliestBlockNumber  = BlockNumber(0) // how are we thinking about having two genesis blocks?
```

Comment:
The "special" genesis block number is just 0, it's not really special / doesn't have special handling in most cases. I think we can effectively start geth with a `genesis.json` with a state root that matches the DB contents and a non-zero block number. So we would still have one true genesis block, but it would be served by the historical RPC service just like any other block older than the block in the init config (not to call it genesis).